core/src/main/java/com/fincity/saas/core/functions/storage/ReadPageStorageObject.java

```diff
@@ -185,4 +200,27 @@ protected Mono<FunctionOutput> internalExecute(ReactiveFunctionExecutionParamete
 				});
 	}
 
+	private Sort getSortObj(JsonArray sortJson){
+
+		if(sortJson.size() == 0) {
+			return Sort.by(Order.desc("updatedAt"));
+		}
+		
+		List<Map<String, String>> sortList = gson.fromJson(sortJson, List.class);
```

Comment:
You should not be using gson.fromjson when you have a json array. JSON object usually will have JSON Elements get them as objects. Change the names "field" and "sortType" to the fields in the Sort.Order class.